docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:16.04
+
+# Update
+RUN apt-get update && apt-get dist-upgrade -y
+
+# Install the necessary packages.
+RUN apt-get install -y \
+    supervisor apache2 git php7.0 libapache2-mod-php7.0 php7.0-cli \
+    php7.0-mysqlnd php7.0-xml php7.0-mbstring php7.0-curl php7.0-zip
```

Comment:
Why do we need php7.0-mysqlnd here? Laravel uses the PDO driver, and mysqlnd shouldn't be used.